node/mdSchema.ts

```diff
@@ -190,6 +199,14 @@ export const schemas = [
           type: 'array',
           title: 'Default Price Tables',
         },
+        organizationCustomFields: {
```

Comment:
These fields are obsolete with the switch to Vbase, we don't need to add them back in

Reply:
My bad, removed them